components/resources/brave_components_resources.grd

```diff
@@ -18,6 +18,10 @@
       <!-- TODO: move to brave_adblock_ui component -->
       <include name="IDR_BRAVE_ADBLOCK_HTML" file="../brave_adblock_ui/brave_adblock.html" type="BINDATA" />
 
+      <!-- WebUI playlist resources -->
+      <!-- TODO: move to playlist component -->
```

Comment:
shouldn't it just live in `components/resources/playlist_resources.grdp`? We did it for Speedreader